### PR TITLE
home work done

### PR DESCRIPTION
**Pull request by RedMasta1988**

### CODE REVIEW

File: lazy_if.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+print(result_1)
+result_2 = num_1 or num_2 or num_3 or "Введены все нули!"
+print(result_2)
+if num_1 > num_2 + num_3:
+    print(num_1 - num_2 - num_3)
+if num_1 < num_2 + num_3:
+    print(num_2 + num_3 - num_1)
+if num_1 > 50 and (num_2 > num_1 or num_3 > num_1):
+    print("Вася")
+if num_1 > 5 or (num_2 == 7 and num_3 == 7):
```
Comment (dvk-net) [inline lazy_if.py]: () arent needed here...

**Review by dvk-net — APPROVED** [review]
good

**Comment by RedMasta1988** [comment]
Very well

File: lazy_if.py (hunk 1 above)
Comment (RedMasta1988) [reply lazy_if.py]: Ok

**Pull request merged**